src/lavinmq/exchange.cr

```diff
@@ -1 +1,11 @@
 require "./exchange/*"
+
+module LavinMQ
+  struct EmptyIterator(T)
+    include Iterator(T)
+
+    def next
+      stop
+    end
+  end
+end
```

Comment:
I was thinking that we could monkey patch stdlib iterator to have the empty method. If my PR is accepted we can just remove that patch and the method from stdlib will be used.

Reply:
Ok, I will try that

Reply:
Done https://github.com/cloudamqp/lavinmq/pull/786/commits/af69bfdb80182c1e1781f52655d3c466f17ff67b

Reply:
and https://github.com/cloudamqp/lavinmq/pull/786/commits/f55e03a8f9d2dd685314affc0d00d2aab66fca22 :)

Reply:
Coolio!